### PR TITLE
[Keyboard] Add aeroboard

### PR DESCRIPTION
**Pull request by EasonQian1**
## Description
add aeroboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/eason/aeroboard/rules.mk [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+SLEEP_LED_ENABLE = yes
+WS2812_DRIVER = spi
+
+DEFAULT_FOLDER = eason/aeroboard
+
+
+
```
Comment (zvecr) [inline keyboards/eason/aeroboard/rules.mk:25]:
```suggestion

```

File: keyboards/eason/aeroboard/rules.mk [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+BOOTLOADER = stm32duino
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
```
Comment (zvecr) [inline keyboards/eason/aeroboard/rules.mk:14]:
```suggestion
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/eason/aeroboard/config.h [hunk 3]
```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Eason
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 16
+
```
Comment (waffle87) [inline keyboards/eason/aeroboard/config.h:9]:
```suggestion
```

File: keyboards/eason/aeroboard/config.h [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ *
+ * NKRO may prevent your keystrokes from being detected in the BIOS, but it is
+ * fully operational during normal computer usage.
+ *
+ * For a less heavy-handed approach, enable NKRO via magic key (LShift+RShift+N)
+ * or via bootmagic (hold SPACE+N while plugging in the keyboard). Once set by
+ * bootmagic, NKRO mode will always be enabled until it is toggled again during a
+ * power-up.
+ *
+ */
```
Comment (waffle87) [inline keyboards/eason/aeroboard/config.h:53]:
```suggestion
```

File: keyboards/eason/aeroboard/aeroboard.h [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ #define ___ KC_NO
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_all( \
```
Comment (waffle87) [inline keyboards/eason/aeroboard/aeroboard.h:18]: (and everywhere else `LAYOUT_all` was used eg. `info.json` & keymaps)
```suggestion
#define LAYOUT( \
```

File: keyboards/eason/aeroboard/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = STM32F103
+
+# Bootloader selection
+BOOTLOADER = stm32duino
+
```
Comment (waffle87) [inline keyboards/eason/aeroboard/rules.mk:6]:
```suggestion
```

File: keyboards/eason/aeroboard/info.json [hunk 7]
```diff
@@ -0,0 +1,89 @@
+{
+    "keyboard_name": "AeroBoard",
+    "manufacturer": "Eason",
+    "url": "",
+    "maintainer": "Eason",
+    "usb": {
+        "vid": "0x8954",
+        "pid": "0x8870",
+        "device_version": "0.0.1"
+    },
```
Comment (waffle87) [inline keyboards/eason/aeroboard/info.json:10]:
```suggestion
    },
    "processor": "STM32F103",
    "bootloader": "stm32duino",
```

File: keyboards/eason/aeroboard/readme.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+# aeroboard
+
+** A customizable hotswap 70% keyboard.
```
Comment (waffle87) [inline keyboards/eason/aeroboard/readme.md:3]:
```suggestion
A customizable hotswap 70% keyboard.
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by EasonQian1** [comment]
@waffle87 Hello，Dude.I have changed it correctly.

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by EasonQian1** [comment]
May I ask whether the PR can be merged
@drashna @fauxpark @waffle87 @zvecr

**Pull request merged**